src/ecdsa/robust_ecdsa/mod.rs

```diff
@@ -0,0 +1,30 @@
+pub mod presign;
+pub mod sign;
+#[cfg(test)]
+mod test;
+
+use crate::ecdsa::{AffinePoint, KeygenOutput, Scalar};
+use serde::{Deserialize, Serialize};
+
+/// The arguments needed to create a presignature.
+#[derive(Debug, Clone)]
+pub struct PresignArguments {
+    /// The output of key generation, i.e. our share of the secret key, and the public key package.
+    /// This is of type KeygenOutput<Secp256K1Sha256> from Frost implementation
+    pub keygen_out: KeygenOutput,
+    /// The desired threshold for the presignature, which must match the original threshold
+    pub threshold: usize,
+}
+
+// The output of the presigning protocol.
+/// Contains the signature precomputed parts performed
+/// independently of the message
```

Comment:
```suggestion
/// Contains the signature parts precomputed
/// independently of the message
```
maybe "shares" instead of "parts" ?